### PR TITLE
New device ZBMicro is added.

### PR DESCRIPTION
**Pull request by liubotongs**
Added Sonoff manufacturer's new device ZBMicro.

### CODE REVIEW

File: src/devices/sonoff.ts [hunk 1]
```diff
@@ -865,6 +926,26 @@ const definitions: Definition[] = [
...
+        extend: [
+            onOff(),
+            binary({
+                name: 'rf_turbo_mode',
+                cluster: 0xFC11,
+                attribute: {ID: 0x0012, type: 0x29},
+                zigbeeCommandOptions: {manufacturerCode: 0x1286},
+                description: 'Enable/disable Radio power turbo mode',
+                valueOff: ['DISABLE', 0x09],
+                valueOn: ['ENABLE', 0x14],
```
Comment (Koenkk) [inline src/devices/sonoff.ts:944]:
```suggestion
                valueOff: [false, 0x09],
                valueOn: [true, 0x14],
```

Comment (liubotongs) [reply src/devices/sonoff.ts:944]: I see that there is no ZBMicro content in the newly merged version, what went wrong?

Comment (Koenkk) [reply src/devices/sonoff.ts:944]: This review comment still needs to be solved before merging, can you take a look at it?

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**